hrms/payroll/doctype/salary_slip/salary_slip.py

```diff
@@ -1858,3 +1856,12 @@ def date_range(start=None, end=None):
 		days = [str(start + timedelta(days=i)) for i in range(delta.days + 1)]
 		return days
 	return []
+
+
+def throw_error_message(row, error, title):
+	doclink = get_link_to_form(row.parenttype, row.parent)
+	message = f""" {title} in <b>{row.parenttype}</b>: {doclink} </a> at row {row.idx} """
+	if error:
+		message += f""" <br><br> Error: {error} """
+
+	frappe.throw(_(message), title=_(title))
```

Comment:
This will break translation. Check https://frappeframework.com/docs/v14/user/en/translations#1-literal-strings